src/api/spec/bootstrap/features/webui/projects_spec.rb

```diff
@@ -27,4 +27,78 @@
       expect(find(:css, '#description-text')).to have_text(very_long_description)
     end
   end
+
+  scenario 'changing project title and description' do
+    login user
+    visit project_show_path(project: project)
+
+    click_on('Edit Project')
+    expect(page).to have_text('Edit Project')
+
+    fill_in 'project_title', with: 'My Title hopefully got changed'
+    fill_in 'project_description', with: 'New description. Not kidding.. Brand new!'
+    click_button 'Accept'
+
+    visit project_show_path(project: project)
+    expect(find(:id, 'project-title')).to have_text('My Title hopefully got changed')
+    expect(find(:id, 'description-text')).to have_text('New description. Not kidding.. Brand new!')
+  end
+
+  describe 'branching' do
+    let(:other_user) { create(:confirmed_user, login: 'other_user') }
+    let!(:package_of_another_project) { create(:package_with_file, name: 'branch_test_package', project: other_user.home_project) }
+
+    before do
+      login user
+      visit project_show_path(project)
+      click_link('Branch Existing Package')
+    end
+
+    scenario 'an existing package to an invalid target package or project' do
+      fill_in('linked_project', with: other_user.home_project_name)
+      fill_in('linked_package', with: package_of_another_project.name)
+      fill_in('Branch package name', with: 'something/illegal')
+      # This needs global write through
```

Comment:
?

Reply:
I think this was copy-pasted from somewhere